mmomni/ansible/playbooks/jitsi/jicofo.config

```diff
@@ -0,0 +1,28 @@
+# Jitsi Conference Focus settings
+# sets the host name of the XMPP server
+JICOFO_HOST=localhost
+
+# sets the XMPP domain (default: none)
+JICOFO_HOSTNAME={{ jitsi_fqdn }}
+
+# sets the secret used to authenticate as an XMPP component
+JICOFO_SECRET={{ jitsi_focus_secret }}
+
+# sets the port to use for the XMPP component connection
+JICOFO_PORT=5347
```

Comment:
Note for the future: the next stable version will no longer use component connections but a client connection.